### ISSUE TITLE
language "et" not supported: Docutils-generated text will be in English

### ISSUE THREAD

**Issue by lsaffre** [sphinx-doc/sphinx#3806]
I have two Sphinx sites in Estonian (et), but Sphinx 1.6 says above warning. In earlier versions there was no problem and everything worked (for me) as expected. Example of such a site is http://belglane.saffre-rumma.net/

What can I do to help get Estonian language back?

**Comment by jfbu** [comment]
Sorry for inconvenience. The warning is generated by Docutils and is also under investigation at #3788. It will happen for any language code not natively known to Docutils (in ``docutils.languages``) Do your builds exhibit any kind of problem or wrong output (in particular regarding rendering of ``"`` or ``'`` quotes) besides displaying the warning itself ?

**Comment by jfbu** [comment]
can you check the fix at https://github.com/sphinx-doc/sphinx/issues/3788#issuecomment-304234520 which consists of adding to the `conf.py` some monkey patch?

another fix is in the works: #3808

**Comment by lsaffre** [comment]
Thanks, Jean-François, that worked for me!

BTW I simplified the code to the following (everything at the end of my *conf.py* file):

```
  import docutils
  from docutils.languages import en

  def patched_get_language(language_code, reporter=None):
      return en

  def setup(app):
      docutils.languages.get_language = patched_get_language

```
Luc

**Comment by jfbu** [comment]
thanks for the confirmation and simplification. You will perhaps observe that smart quotes are applied (transformation of `"` and `'` into suitable curly quotes ``„“‚‘`` for Estonian), but if PR #3808 which fixes in another way your issue is merged, then the smart quotes will not be applied (with Docutils 0.13.1), at least in current state.

**Comment by tk0miya** [comment]
Fixed at #3813
